.circleci/workflows.yml

```diff
@@ -48,85 +80,110 @@ parameters:
+            - aws-s3/sync:
+                arguments: |
+                  --endpoint-url https://cellar-c2.services.clever-cloud.com \
+                  --acl public-read
+                from: ~/release/
+                to: "s3://gravitee-releases-downloads/" 
+  restore-maven-job-cache:
+    description: Restore Maven cache for a dedicated job
+    parameters:
+      jobName:
+        description: The job name
+        type: string
+        default: ""
+    steps:
+      - restore_cache:
+          keys:
+            - gravitee-access-management-v1-<< parameters.jobName >>-{{ .Branch }}-{{ checksum "pom.xml" }}
+            - gravitee-access-management-v1-<< parameters.jobName >>-{{ .Branch }}-
+            - gravitee-access-management-v1-<< parameters.jobName >>-
+  save-maven-job-cache:
+    description: Restore Maven cache for a dedicated job
+    parameters:
+      jobName:
+        description: The job name
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: "Exclude all AM artefacts from cache."
+          command: rm -rf ~/.m2/repository/io/gravitee/am
+      - save_cache:
+          paths:
+            - ~/.m2
+          key: gravitee-access-management-v1-<< parameters.jobName >>-{{ .Branch }}-{{ checksum "pom.xml" }}
+          when: always
+  notify-on-failure:
+    steps:
+      - keeper/env-export:
+          secret-url: keeper://ZOz4db245GNaETVwmPBk8w/field/password
+          var-name: SLACK_ACCESS_TOKEN
+      - slack/notify:
+          channel: C0336B2E96K
+          event: fail
+          template: basic_fail_1
+  notify-on-success:
     steps:
-      - snyk/scan:
-          organization: gravitee.io
-          monitor-on-build: false
-          severity-threshold: critical
-          docker-image-name: ${IMAGE_TAG}
-          fail-on-issues: false
-          additional-arguments: --app-vulns
+      - keeper/env-export:
+          secret-url: keeper://ZOz4db245GNaETVwmPBk8w/field/password
+          var-name: SLACK_ACCESS_TOKEN
+      - slack/notify:
+          channel: C0336B2E96K
+          event: pass
+          template: basic_success_1
+  add-docker-image-in-snyk:
+    parameters:
+      docker-image-name:
+        type: string
+      version:
+        type: string
+    steps:
+      - run:
+          name: Add << parameters.docker-image-name >> << parameters.version >> to Snyk
+          command: |
+              curl --fail \
+                   --include \
+                   --request POST \
+                   --header "Content-Type: application/json; charset=utf-8" \
+                   --header "Authorization: token ${SNYK_API_KEY}" \
+                   --data-binary "{
+                \"target\": {
+                  \"name\": \"graviteeio/<< parameters.docker-image-name >>:<< parameters.version >>\"
+                }
+              }" \
+              "https://api.snyk.io/api/v1/org/${SNYK_ORG_ID}/integrations/${SNYK_INTEGRATION_ID}/import"
   notify-snyk-result:
```

Comment:
This command may be useless now as snyk scan isn't perform during the `publish-images-azure-registry` job.
Maybe I'm wrong